folding/store.py

```diff
@@ -169,9 +169,20 @@ def to_series(self):
     def to_frame(self):
         return pd.DataFrame([self.to_series()])
 
-    def update(self, loss: float, hotkey: str, commit_hash: str, gro_hash: str):
+    def update(
+        self,
+        loss: float,
+        hotkey: str,
+        commit_hash: str,
+        gro_hash: str,
+        hotkeys: List[str] = None,
```

Comment:
i am not sure if I like this name... what it really is, is the hotkeys that you want to be associated to the job. So at any time, you can re-assign the job to any hotkeys. Maybe `job_hotkeys`? `reset_hotkeys`?